docs/UserGuide.md

```diff
@@ -158,147 +128,241 @@ Examples:
 
-Format: `find KEYWORD`
+Format: `addLesson -name NAME [-start HH:MM] [-end HH:MM] [-day YYYY/MM/DD] [-subject SUBJECT]` <br>
+Format: `addLesson -name NAME [-start HH:MM] [-end HH:MM] [-day YY/MM/DD] [-subject SUBJECT]`<br>
+Format: `addLesson -name NAME [-start HH:MM] [-end HH:MM] [-day MM/DD] [-subject SUBJECT]` <br>
+Note: If no year is specified, the year is assumed to be current year.
+<box type="tip" seamless>
 
-1. In `STUDENTS` list:
-   * The search is case-insensitive. e.g `hans` will match `Hans`
-   * Only the name is searched.
-   * Persons matching part of the keyword will be returned (i.e. `OR` search).
-     e.g. `Hans` will return `Hanso Gruber`, `Lee Hansel`
-2. In `SCHEDULE` list:
-    * The search is case-insensitive. e.g `lesson` will match `Lesson`
-    * Only the name is searched.
-    * Lessons matching part of the keyword will be returned (i.e. `OR` search).
-      e.g. `Lesson Chem` will return `Lesson Chemistry`, `Bishan Lesson Chem`
+**Tips:**
+- A new lesson cannot have the same name as existing lessons in the schedule list.
+- A new lesson cannot have overlapping timings with existing lessons in the schedule list.
+- Start time cannot be after end time.
+- If the user is currently in list `SCHEDULE`, the command can be shortened to `add`
+- Only one subject can be assigned to a lesson.
+- Subjects can only be MATHEMATICS, PHYSICS, BIOLOGY, CHEMISTRY or ENGLISH.
+  </box>
 
-Example Success Output:
+Examples:
+* `addLesson -name Chemistry Lesson at Bishan -start 14:00 -end 15:00 -day 2023/12/12 -subject MATHEMATICS`
+* `addLesson -name Lesson at Tai Seng -start 09:00 -end 11:00 -day 03/21 -subject PHYSICS`
+* In list `SCHEDULE`:
+    * `add -name Lesson at Tai Seng -start 09:00 -end 11:00 -day 03/21 -subject PHYSICS`
+
+Success Output:
 ```
-1 persons listed!
-2 lessons listed!
+New lesson added: Lesson Chemistry Lesson at Bishan from 2:00 PM to 3:00 PM on 12-12-2023
 ```
-
 Failure Output:
 ```
-0 persons listed!
-0 lessons listed!
+Exist lesson clashes with this lesson: Lesson Chemistry Lesson at Bishan from 2:00 PM to 3:00 PM on 12-12-2023
 ```
+### Editing a lesson : `editLesson`
 
-### Showing a student's details : `show`
-
-The show command has different behaviours depending on the current list:
-
-1. In `STUDENTS` list:
-   - Shows the details of the specified student from the contact list in the application.
-2. In `SCHEDULE` list:
-   - Shows the details of the specified lesson from the schedule list in the application.
-3. In `TASKS` list:
-   - Shows the details of the specified task from the full task list in the application.
-
-Format: `show INDEX`
+Edits an existing lesson in the application.
 
-* Shows the details of the student/lesson/task at the specified `INDEX`.
-* The index refers to the index number shown in the displayed student/schedule/task list.
+Format: `editLesson INDEX [-name NAME] [-start HH:MM] [-end HH:MM] [-day YYYY/MM/DD] [-subject SUBJECT]` <br>
```

Comment:
Same issue over day here;

Reply:
Made the necessary changes in recent commits. Thank you!

Reply:
Thank you!
Do not need to reply message one by one lah, I feel bad causing you extra trouble doing so.